flepimop/gempyor_pkg/src/gempyor/utils.py

```diff
@@ -346,3 +353,164 @@ def bash(command):
     print("------------")
     print(f"lsblk: {bash('lsblk')}")
     print("END AWS DIAGNOSIS ================================")
+
+
+def create_resume_out_filename(filetype: str, liketype: str) -> str:
+    run_id = os.environ.get("FLEPI_RUN_INDEX")
+    prefix = f"{os.environ.get('FLEPI_PREFIX')}/{os.environ.get('FLEPI_RUN_INDEX')}"
+    inference_filepath_suffix = f"{liketype}/intermidate"
+    inference_filename_prefix = "{:09d}.".format(int(os.environ.get("FLEPI_SLOT_INDEX")))
+    index = "{:09d}.{:09d}".format(1, int(os.environ.get("FLEPI_BLOCK_INDEX")) - 1)
+    extension = "parquet"
+    if filetype == "seed":
+        extension = "csv"
+    return file_paths.create_file_name(
+        run_id=run_id,
+        prefix=prefix,
+        inference_filename_prefix=inference_filename_prefix,
+        inference_filepath_suffix=inference_filepath_suffix,
+        index=index,
+        ftype=filetype,
+        extension=extension,
+    )
+
+
+def create_resume_input_filename(filetype: str, liketype: str) -> str:
+    run_id = os.environ.get("RESUME_RUN_INDEX")
+    prefix = f"{os.environ.get('FLEPI_PREFIX')}/{os.environ.get('RESUME_RUN_INDEX')}"
+    inference_filepath_suffix = f"{liketype}/final"
+    index = os.environ.get("FLEPI_SLOT_INDEX")
+    extension = "parquet"
+    if filetype == "seed":
+        extension = "csv"
+    return file_paths.create_file_name(
+        run_id=run_id,
+        prefix=prefix,
+        inference_filepath_suffix=inference_filepath_suffix,
+        index=index,
+        ftype=filetype,
+        extension=extension,
+    )
+
+
+def get_parquet_types_for_resume() -> List[str]:
```

Comment:
This is very good, but these functions should take arguments and not read environment variables, because we want to use them from gempyor (and have less and fewer environment variable), and because if we need an environment variable version we can add a wrapper around it.

(think of a script like simulate.py, that is e.g called resume_from.py : this script parse the environment variable from the safety of the click module and call these functions.

Reply:
Another note: seeding is a csv, so I think this function should be renamed "get_filetype_for_resume" (this 4 letter id is called filetype throughout the code)-